kornia/augmentation/augmentation3d.py

```diff
@@ -144,8 +147,11 @@ def __init__(
     def __repr__(self) -> str:
         return self.__class__.__name__ + f"({super().__repr__()})"
 
+    def generate_parameters(self, batch_shape: torch.Size):
+        return dict(batch_shape=batch_shape)
```

Comment:
If it is the case, I think we shall move it into the parent class.

Reply:
I don't understand why to move it there. Not all transforms need this info. I'm not against it.

Reply:
I avoided to move into the parent class. This line is repeated 4 times (horizontal & vertical flip for 2D and 3D cases). To avoid repetition, it should be moved to `_BasicAugmentationBase`. Currently, it returns an empty dictionary and no other transforms call to `super().generate_parameters`. So, I don't see how could be useful to put specific code for flips in parent classes.